sydent/sydent.py

```diff
@@ -355,6 +326,42 @@ def parse_config_file(config_file):
     return cfg
 
 
+def setup_logging(cfg):
+    log_format = (
+        "%(asctime)s - %(name)s - %(lineno)d - %(levelname)s"
+        " - %(message)s"
+    )
+    formatter = logging.Formatter(log_format)
+
+    logPath = cfg.get('general', "log.path")
+    if logPath != '':
+        handler = logging.handlers.TimedRotatingFileHandler(
+            logPath, when='midnight', backupCount=365
+        )
+        handler.setFormatter(formatter)
+
+        def sighup(signum, stack):
+            logger.info("Closing log file due to SIGHUP")
+            handler.doRollover()
+            logger.info("Opened new log file due to SIGHUP")
+    else:
+        handler = logging.StreamHandler()
+
+    handler.setFormatter(formatter)
+    rootLogger = logging.getLogger('')
+    rootLogger.setLevel(cfg.get('general', 'log.level'))
+    rootLogger.addHandler(handler)
+
+    observer = log.PythonLoggingObserver()
+    observer.start()
+
+
+def getConfigFilePath():
```

Comment:
I know there's a bunch of camel case in sydent already, but let's not exacerbate the problem by adding more.
```suggestion
def get_config_file_path():
```